contrib/kubespray/roles/docker-cache/install/tasks/add-docker-cache-config.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: copy add_docker_cache_config.py script into /tmp
+  copy:
+    src: add_docker_cache_config.py
+    dest: /tmp/add_docker_cache_config.py
+
+- name: run add_docker_cache_config.py script to add docker cache config
+  ansible.builtin.command:
+    argv:
+      - sudo
```

Comment:
I notice there's "become: true" already. So you can remove this sudo.

Reply:
ok, I will update this